app/views/professors/academics/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= simple_form_for [:professors, @academic] do |f| %>
+  <%= f.error_notification %>
+  <%= f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present? %>
```

Comment:
The lines 2 and 3 can be removed. This way it shows two messages: 

![image](https://user-images.githubusercontent.com/22449115/52181618-e2076500-27da-11e9-9cfa-9015c9fafb39.png)


It's necessary just one, that can be: "Alguns erros foram encontrados, por favor verifique:"